client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx

```diff
@@ -0,0 +1,90 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from 'components/select-dropdown';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+class DisconnectSurvey extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			reasonSelected: 'missingFeature',
+			compactButtons: false,
+			renderFull: false,
+		};
+
+		this.state = initialState;
+	}
+
+	renderFull = () => {
+		// placeholder
+		return (
+			<div>
+				{' '}
+				{ ' follow-up selectable' }{' '}
+			</div>
+		);
+	};
+
+	logReason = option => {
+		this.setState( {
+			reasonSelected: option.value,
+			renderFull: true,
+		} );
+	};
+
+	render() {
+		const { translate, siteSlug } = this.props;
+		const { reasonSelected, compactButtons, renderFull } = this.state;
+
+		const textShareWhy =
+			translate( 'Would you mind sharing why you want to' + ' disconnect ' ) +
+			`${ siteSlug }` +
+			translate( ' from Wordpress.com ' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Updates from WordPress.com' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/site-settings-disconnect-site-survey)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).